capsules/src/process_console.rs

```diff
@@ -561,6 +640,32 @@ impl<'a, A: Alarm<'a>, C: ProcessManagementCapability> ProcessConsole<'a, A, C>
                     Ok(s) => {
                         let clean_str = s.trim();
 
+                        // Try to add a new command to the history buffer
+                        self.command_history.map(|cmd_arr| {
```

Comment:
To improve code size, if the user `COMMAND_HISTORY_LEN` as 0, this should not be added to the compiled code. Placing this in an `if` will allow the compiler to optimize the code and delete this.

Reply:
Done